package.json

```diff
@@ -36,7 +36,7 @@
     "initialize": "yarn clean && git submodule update --init && yarn install --unsafe-perm --frozen-lockfile && yarn install:deploy && yarn compile:contracts",
     "build": "yarn workspace ui run build",
     "lint": "yarn wsrun --exclude token-bridge-contracts lint",
-    "test": "yarn wsrun --exclude monitor --exclude oracle-e2e --exclude ui-e2e --exclude monitor-e2e test",
```

Comment:
why was monitor excluded before?

Reply:
The `test` command is to run the unit tests. The monitor has e2e tests but it didn't have unit tests so far, that's why it was excluded. I added the firsts unit tests for some methods here  https://github.com/poanetwork/tokenbridge/pull/223/commits/a0d6f20ae0cbd74c951af955c4428a16e86fed29

Reply:
thanks for clarification